app/assets/stylesheets/components/navigation.scss

```diff
@@ -112,4 +112,26 @@
       }
     }
   }
+
+  &--item {
+    padding: $su-3;
+    text-align: center;
+    width: 100%;
+
+    &.selected {
```

Comment:
In most cases, wherever we have this "selected/current" state we use "current" keyword. 
Also, since we are all-in with BEM naming convention, it should probably be `.crayons-tabs--item--selected`.

Reply:
Noted! This is helpful because a lot of the existing markup/class names use "selected" to indicate status. I'll update this to `current`.